public/js/dashboardCategories.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * TODO: GET all categories from database
+ * @return JSON of all category entry information 
+ */
+function getAllCategories() {
+    var allCategories = [
+        {
+            "ID": "clothing",
+            "name": "Clothing",
+        }, {
+            "ID": "food",
+            "name": "Food",
+        }, {
+            "ID": "house-supplies",
+            "name": "Household Supplies",
+        }
+    ]
+    return allCategories;
+}
+
+/**
+ * TODO: GET an organizations accepted cagtegories from database
```

Comment:
nit: move TODO from header to function body (potentially line 26). Since this is function implementation details and not function level change.